samples/cellular/modem_shell/src/utils/mosh_print.c

```diff
@@ -180,11 +182,7 @@ void mosh_print_version_info(void)
 	/* shell_print() is not used here, because this function is called early during
 	 * application startup and the shell might not be ready yet.
 	 */
-#if defined(APP_VERSION)
-	printk("\nMOSH version:       %s\n", STRINGIFY(APP_VERSION));
-#else
-	printk("\nMOSH version:       unknown\n");
-#endif
+	printk("\nMOSH version:       %s-%s\n", NCS_VERSION_STRING, NCS_COMMIT_STRING);
```

Comment:
It would be nice to have letter 'v' before the version number to be aligned with the Zephyr and SDK version strings.

Reply:
Done this to modem_info, mosh (2 places) and desh.